#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_gallery_extension_install_hook.dpatch by Tim Retout <tim@retout.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Respect the value of $(DESTDIR) in custom install hook.

@DPATCH@
diff -urNad f-spot~/extensions/GalleryExport/Makefile.am f-spot/extensions/GalleryExport/Makefile.am
--- f-spot~/extensions/GalleryExport/Makefile.am	2008-03-09 01:06:51.000000000 +0000
+++ f-spot/extensions/GalleryExport/Makefile.am	2008-03-09 01:08:22.000000000 +0000
@@ -36,7 +36,7 @@
 plugindir = $(pkglibdir)/extensions
 
 install-data-hook:
-	rm -f $(plugindir)/GalleryExport.addin.xml
+	rm -f $(DESTDIR)$(plugindir)/GalleryExport.addin.xml
 
 plugin_DATA =			\
 	$(PLUGIN_ASSEMBLY)
